### DISCUSSION TITLE
Temporal Data set from TUDataset

### DISCUSSION THREAD

**Discussion by xavierallem** [pyg-team/pytorch_geometric#9323] [Q&A]
I am trying to use 

>current_dir = os.getcwd()

> path = osp.join(current_dir, '../../Data/INFECTIOUS')

`dataset` = TUDataset(path, name='infectious_ct1',use_node_attr=True,use_edge_attr=True).shuffle()

and I get the following error

> ValueError: expected sequence of length 4 at dim 1 (got 2)

Is it fixed? I have seen an issue dating to 2020.

**Comment by rusty1s** [comment]
Got it. I think this is because this is a temporal graph which we cannot read yet. Moving this to an issue.
